modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/running-gitops-control-plane-workloads-on-infrastructure-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
+
+.Prerequisites
+* You have installed the {gitops-title} Operator in your cluster.
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Label an existing node as infrastructure node by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/infra=
+----
++
+.Example output
+[source,terminal]
+----
+node/<node_name> labeled
+----
+
+. Edit the {gitops-title} `Subscription` resource:
++
+[source,terminal]
+----
+$ oc -n openshift-gitops-operator edit subscription openshift-gitops-operator
+----
+
+. Add `nodeSelector` and `tolerations` to the `spec.config` field in the `Subscription` resource:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-gitops-operator
+spec:
+  config:
+    nodeSelector: <1>
+      node-role.kubernetes.io/infra: ""
+    tolerations: <2>
+    - key: node-role.kubernetes.io/infra
+      operator: Exists
+      effect: NoSchedule
+----
+<1> This ensures that the operator pod is only scheduled on an infrastructure node.
+<2> This ensures that the pod is accepted by the infrastructure node.
++
+.Example output
+[source,terminal]
```

Comment:
```suggestion
[source,text]
```

Reply:
As per previous comment, I will not make this change. Thank you!